File: librarySystem/Video.cpp
#include <iostream>
#include "Video.h"
#include <sstream>

using namespace std;

 Video::Video(string &cno, string &titl, string &sub, string &desc, string &dist, string &note, string &ser, string &lbl)
        : Media(cno, titl, sub, note), description(desc), distributor(dist), series(ser), label(lbl)
 {
 }


Video::~Video()
{
}

void Video::display() const
{
	cout<<"--------------Video Details---------------"<<endl;
	Media::display();
	cout<<"Description------------"<<this->description<<endl;
	cout<<"Distributor------------"<<this->distributor<<endl;
	cout<<"Notes------------------"<<this->notes<<endl;
	cout<<"Series-----------------"<<this->series<<endl;
	cout<<"Label------------------"<<this->label<<endl;
	cout<<"------------------------------------------"<<endl;
}

bool Video::compare_other(const string &other) const
{
	if((this->description.find(other) != std::string::npos) ||
		(this->notes.find(other) != std::string::npos) ||
		(this->distributor.find(other) != std::string::npos))
	{
		return true;
	}
	else
		return false;
}

File: libPrjPython/Video.py
from Media import Media

class Video(Media):
    
    def __init__(self, callno, title, subject, description, distributor, notes, series, label):
        super(Video, self).__init__(callno, title, subject, notes)
        self.description = description
        self.distributor = distributor
        self.series = series
        self.label = label
    
    def display(self):
       print "---------------Video Details-------------------------------------------------------------------------"
       super(Video, self).display()
       print "Description          : ", self.description
       print "Distributor          : ", self.distributor
       print "Notes                : ", self.notes
       print "Series               : ", self.series
       print "Label                : ", self.label
       print "\n"

    def compare_other(self,srch_other):
        return srch_other in self.description or srch_other in self.notes or srch_other in self.distributor 
